Data/Sys/GameSettings/RM3.ini

```diff
@@ -19,6 +19,7 @@ EmulationIssues =
 
 [Video_Settings]
 SafeTextureCacheColorSamples = 512
+UseRealXFB = True # Prevent black bar
```

Comment:
You want to enforce real xfb to metroid prime 3 for the black bar issue? Please add:
UseXFB = True
To enforce virtual xfb instead and set the line you added to false.
The way you did it was also false, see my thread of gameini settings in the forums on why real and virtual xfb settings should be used always together in the gameinis, the way you did it here if someone enabled virtual xfb in his general settings it would switch to real xfb otherwise nothing would happen.